### DISCUSSION TITLE
What features are included in Pro license?

### DISCUSSION THREAD

**Discussion by majuss** [tiny-pilot/tinypilot#1018] [Q&A] [answered]
Is H264 already available? Can I install Tinypilot Pro via Docker? 

These are the only two features I care about, but I can't find any information if they are included in Pro.

**Comment by mtlynch** [comment, marked as answer]
Thanks for reaching out, @majuss!

>Is H264 already available?

H264 is available as an [opt-in experimental feature](https://tinypilotkvm.com/blog/whats-new-in-2022-05), but it's not fully integrated into the web UI. So you can view video over H264/WebRTC in the browser, but you can't control H264 specific settings yet.

>Can I install Tinypilot Pro via Docker?

No, unfortunately, TinyPilot Pro is not available via Docker. I love Docker and we've considered supporting it, but it's nontrivial for an application like TinyPilot, as TinyPilot requires access to hardware features on the host.

A user shared [a proof-of-concept](https://github.com/tiny-pilot/tinypilot/issues/403#issuecomment-761405632) for running TinyPilot Community in a Docker container, but we haven't integrated those changes.

Can you share more about why you'd like to run it under Docker?

**Comment by majuss** [comment]
I would like to run tinypilot on Home Assistant Operating System. I don't like multiple pis which consume energy.

Is H264 really only working on Debian 10 buster? Was it tested with Bullseye?

**Comment by majuss** [comment]
https://github.com/tiny-pilot/tinypilot/blob/master/Dockerfile

So Dockerimage is coming? @mtlynch

**Reply by mtlynch** [reply]
@majuss - No, unfortunately not. That Dockerfile is for a DevOps task, not for running TinyPilot within Docker.
